dashboard/app/views/levels/_apps_dependencies.html.haml

```diff
@@ -55,6 +56,10 @@
     %script{src: webpack_asset_path('js/blockly.js')}
     %script{src: webpack_asset_path("js/#{js_locale}/blockly_locale.js")}
 
+-# ailab external repo dependencies
+- if use_ailab
+  %script{src: webpack_asset_path("js/#{js_locale}/mlPlayground_locale.js")}
```

Comment:
In `ailab` levels, it is necessary to add `mlPlayground_locale.js` to make translations available. By default, only the `#{app}_locale.js` is added.
Therfore for ailab levels, `ailab_locale.js` and `mlPlayground_locale.js` will be available.